EZSwiftExtensionsTests/DoubleTests.swift

```diff
@@ -42,6 +42,15 @@ extension DoubleTests {
         XCTAssertNotEqual(Double(anotherDouble.toInt), anotherDouble)
     }
     
+    func testCGFloat() {
+        XCTAssertEqual(double.toCGFloat, 3.14)
+        XCTAssertEqual(otherDouble.toCGFloat, -147.9564)
+        XCTAssertTrue(double.toCGFloat is CGFloat)
```

Comment:
Yes I believe its safe to remove it. Swift is statically and strongly typed for this very reason.